specfile/README.md

```diff
@@ -0,0 +1,75 @@
+# Packit requirements for the spec-file library
+
+## Manipulation
+
+- We create a Python object by specifying a path of the spec-file.
+- (optional) After a change, we can reload the content (can be done by recreating of the object).
+- After the manipulation, the content is synced to the file or, we have a way to do this explicitely.
+- Be able to copy a section of spec-file to another object.
+
+## Version and release
+
+- The version and release fields can be get.
```

Comment:
Do you mean this with macros expanded or the raw values? Or both?

Reply:
This is with what I am really not sure in general -- do we want to have both modes?

For us, it's crucial to be able to do the edits without side effects (the diff after a change is local and minimal).